.github/workflows/CI-notebook.yml

```diff
@@ -41,6 +41,11 @@ jobs:
         run: |
           conda install --yes --quiet pytorch torchvision captum -c pytorch
 
+      - name: build pywinpty from conda
+        shell: bash -l {0}
+        run: |
+          conda install pywinpty
+
```

Comment:
Seems like pywinpty is not available in python 3.8

Run conda install pywinpty
Collecting package metadata (current_repodata.json): ...working... done
Solving environment: ...working... failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working... failed with initial frozen solve. Retrying with flexible solve.

PackagesNotFoundError: The following packages are not available from current channels:

  - pywinpty

Current channels:

Reply:
oooooh, the problem is it can only be installed on windows, I need to put a flag that only installs this when platform==windows

Reply:
changed to only install when platform == windows